lib/views/pages/account_settings/account_settings.dart

```diff
@@ -20,11 +21,32 @@ class AccountSettings extends StatefulHookConsumerWidget {
 class _AccountSettingsState extends ConsumerState<AccountSettings> {
   final String currency = 'PHP';
   final String spendingLimit = '18,000';
+  final SpendingLimitController spendingLimitController =
+      SpendingLimitController();
+
+  String currentSpendingLimit = '0.0';
+
+  @override
+  void initState() {
+    super.initState();
+    _totalValues();
+  }
+
+  Future<void> _totalValues() async {
+    var now = DateTime.now();
+    var monthAfter = DateTime.now().add(Duration(days: 5));
```

Comment:
@AubreyV
      
      maybe we can use this? so that the range is from first day of the month up to the last day.
      
      DateTime now = new DateTime.now();
      DateTime firstDayofMonth = new DateTime(now.year, now.month, 1);
      DateTime lastDayOfMonth = new DateTime(now.year, now.month + 1, 0);